src/features/search/searchSlice.js

```diff
@@ -7,16 +7,18 @@ const searchSlice = createSlice({
     moviesList: null,
     peopleList: null,
     statusMovie: null,
-    statusPeople: null
+    statusPeople: null,
+    page: 1,
```

Comment:
line 11: this page is probably not needed here

Reply:
if it will be work without line 11, we can delete it